tests/vector/utils.py

```diff
@@ -109,6 +109,7 @@ def step(self, action):
 def make_env(env_name, seed):
     def _make():
         env = gym.make(env_name)
+        env.action_space.seed(seed)
```

Comment:
Why seed the action space, but not the observation space?

Reply:
As we are testing that the SyncVectorEnv uses a seeded action space as the one passed by the environment functions then we only need to ensure that the action space is seeded correctly. 
I dont know any code that uses the randomness in the observation space so I haven't set it

Reply:
Fair enough. I guess it could be considered slightly out-of-scope for this PR, even though it makes a lot of sense, in my opinion.

Reply:
This can be marked as resolved.